compiler/noirc_frontend/src/hir_def/types.rs

```diff
@@ -2244,7 +2349,8 @@ fn convert_array_expression_to_slice(
 
 impl BinaryTypeOperator {
     /// Perform the actual rust numeric operation associated with this operator
-    pub fn function(self, a: u32, b: u32) -> Option<u32> {
+    // TODO: the Kind is included since it'll be needed for size checks
```

Comment:
unresolved todo, issue?

Reply:
Ah yeah, need to link [here](https://github.com/noir-lang/noir/pull/6137)